qiskit_optimization/problems/variable.py

```diff
@@ -148,3 +148,13 @@ def as_tuple(self) -> Tuple[str, Union[float, int], Union[float, int], VarType]:
             variable type.
         """
         return self.name, self.lowerbound, self.upperbound, self.vartype
+
+    def __repr__(self):
+        return f"<Variable: {str(self)}>"
+
+    def __str__(self):
+        if self._vartype == VarType.BINARY:
+            return f"{self._name} ({self._vartype.name.lower()})"
+        lowerbound = "" if self._lowerbound == -INFINITY else f"{self._lowerbound} <= "
+        upperbound = "" if self._upperbound == INFINITY else f" <= {self._upperbound}"
+        return f"{lowerbound}{self._name}{upperbound} ({self._vartype.name.lower()})"
```

Comment:
Just a comment. the `as_tuple` method above acts on similar values but does it via the getters  rather than direct access to the instance vars. Was there a specific reason to do this and not use the getters here i.e `self._name` versus `self.name`. I must admit when I have getters I tend to use these and not go direct to the instance vars that may be backing it. Since this is more an internal representation maybe that was the thought in being more direct.

Reply:
I don't have any specific reason to use `self._name` instead of `self.name`. I used `self._name` because `__str__` and `__repr__` are methods and are allowed to access the instance variable directly. I don't have a strong opinion on the choice of `self._name` or `self.name`. I follow your recommendation.

Reply:
I don't have a strong opinion either which is why I started things above with "Just a comment..."

At present the getter simply returns the instance var value unaltered so using the instance var is the exact same; and I guess that's unlikely to change. Ok, lets stay with what you have then.